### PR TITLE
Add coveragerc file

### PR DESCRIPTION
**Pull request by IAlibay**
Let's see if we can force codecov to behave

### CODE REVIEW

**Comment by IAlibay** [comment]
Well at least we know it won't go below 0% 🤣

**Comment by IAlibay** [comment]
So it is a problem with the pip install... how odd

**Comment by IAlibay** [comment]
Right so:

`pip install -v` --> bad
`pip install -e` --> good
`cd panedrlite && python setup.py install` --> bad

**Comment by IAlibay** [comment]
Ok for now I think this is probably the way to go for CI?

With a bit more hands-on time with panedr I agree with @jbarnoud, installing panedrlite should expose a panedr import, otherwise you end up with different behaviour with each package

File: .github/workflows/gh-ci.yaml [hunk 1]
```diff
@@ -54,10 +54,13 @@ jobs:
         conda info
         conda list
 
-    - name: install package
+    - name: install panedrlite package
       run: |
-        python -m pip install -v ./panedrlite
-        python -m pip install -v ./panedr
+        cd panedrlite && python setup.py develop
```
Comment (orbeckst) [inline .github/workflows/gh-ci.yaml:59]: Maybe add a comment why we are _not_ using `pip`?

File: .github/workflows/gh-ci.yaml [hunk 2]
```diff
@@ -45,7 +45,7 @@ jobs:
 
     - name: install package deps
       run: |
-        mamba install pandas pytest pytest-xdist pytest-cov codecov
+        mamba install pandas pytest pytest-xdist pytest-cov codecov pbr
```
Comment (orbeckst) [inline .github/workflows/gh-ci.yaml:48]: oops, good catch — I suppose that was previously installed by pip along the way?

**Review by orbeckst — APPROVED** [review]
Looks as if it's working...

**Pull request merged**